### ISSUE TITLE
Pagination: Empty links

### ISSUE THREAD

**Issue by marcoscv-work** [liferay/clay#5286]
In pagination examples many links can be generated as emptly links without `href`, it is not correct since they cannot get focus.
For example, we could be adding `aria-current="page"` but if the link is not tabbaulable we cann know it is the current and selected page by screen reader.

**Recommendation**:
- Make href mandatory for links
or
- Add a warning if you are creatign an emptly link
